src/Misc.h

```diff
@@ -2,6 +2,8 @@
 
 namespace Misc
 {
+  extern const char * postExitCmd;
```

Comment:
I'd much rather not have this, it looks gross.

Reply:
yeah, I just made the code simple and keep consistent with other code style already in this project, like here:

namespace Renderer
{
  extern const char * defaultShaderFilename;

but indeed I can do it better by offering an interface such as setter/getter

Reply:
The `ExecuteCommand` function itself can internally test if the incoming path is NULL or empty string, that's probably the cleanest way.

Reply:
we need to know the exit cmd script path in other place, for Mac, when tap "Quit" on TouchBar, we need to know where is the script path

so in TouchBar.mm, I made this change which require an access of postExitCmd

- (void)quit {
     Misc::ExecuteCommand(Misc::defaultPostExitCmd, Renderer::defaultShaderFilename);
     exit(0);
 }

Reply:
to make code clean, I just commit this:

[make code clean](https://github.com/Gargaj/Bonzomatic/pull/105/commits/0af97504dabc2d65ae9f80bbbb1cdbe5db5cec55)